src/test/test-openssl.c

```diff
@@ -94,7 +94,7 @@ TEST(ecc_pkey_curve_x_y) {
         assert_se(memcmp_nn(y, y_len, y2, y2_size) == 0);
 }
 
-TEST(invalid) {
+TEST(invalid_pem) {
```

Comment:
this change doesn't seem related to the rest of the commit? (`openssl: add openssl_digest_many()`)

Reply:
sorry! i think this slipped in while i was rebasing/editing; removed it.